data/src/error_analysis/README.md

```diff
@@ -0,0 +1,45 @@
+# Error Analysis tool for G2P
```

Comment:
"Error Analysis" is not a proper noun so it should be "Error analysis" I think.

Reply:
fixed